### PR TITLE
Fix/lfs images for docs

### LINKED ISSUE #482: Timestamps in documentation pages
"Last update" and "created" time displayed at the bottom of documentation pages always coincide. 
image

### LINKED ISSUE #540: Fix rendering of influences plot in readme
With the recent change in #532 that moved all png images to git lfs, the influences plot in the readme no longer renders properly.

image

**EDIT** This could be related to the recent change we made to use absolute urls in the readme.

### PR DESCRIPTION
**Pull request by mdbenito**
### This PR:

- Closes #540 
- Closes #482
- Moves examples images to `examples/img`
- Fixes a broken link
- Adds svgs to lfs

### Checklist

- [ ] ~Wrote Unit tests (if necessary)~
- [x] Updated Documentation (if necessary)
- [x] Updated Changelog
- [ ] ~If notebooks were added/changed, added boilerplate cells are tagged with `"tags": ["hide"]` or `"tags": ["hide-input"]`~

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
+        lfs: true
```
Comment (AnesBenmerzoug) [inline .github/workflows/main.yaml:60]: I think this should also be set in the publish workflow.

**Review by AnesBenmerzoug — DISMISSED** [review]

File: .github/workflows/main.yaml (hunk 1 above)
Comment (mdbenito) [reply .github/workflows/main.yaml:60]: Thx good catch

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -29,11 +29,14 @@ jobs:
       - name: Checking out last commit in release
         if: ${{ github.event_name != 'workflow_dispatch' }}
         uses: actions/checkout@v4
+        with:
```
Comment (AnesBenmerzoug) [inline .github/workflows/publish.yaml:32]: I just noticed that we also need to set `fetch-depth: 0` here in order to make the timestamps work in the documentation.

Comment (mdbenito) [reply .github/workflows/publish.yaml:32]: So that closes #482 ? If you have already tested it, please push to this branch and merge! 🙏🏽

Comment (AnesBenmerzoug) [reply .github/workflows/publish.yaml:32]: Yes, If you check the development version of the docs, you can see that the timestamps were fixed there. I just forgot to also update the publish workflow.

Comment (mdbenito) [reply .github/workflows/publish.yaml:32]: Ok, I'll add it then

**Pull request merged**